test/integration/admin_login_test.rb

```diff
@@ -61,7 +60,7 @@ def setup
     click_link_or_button('Admin Dashboard')
     assert admin_dashboard_path, current_path
 
-    click_link_or_button('Categories')
+    click_link_or_button('Category')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.